pkg/contract_watcher/shared/getter/getter_test.go

```diff
@@ -47,8 +47,7 @@ var _ = Describe("Interface Getter", func() {
 			transactionConverter := rpc2.NewRpcTransactionConverter(ethClient)
 			blockChain := eth.NewBlockChain(blockChainClient, rpcClient, node, transactionConverter)
 			interfaceGetter := getter.NewInterfaceGetter(blockChain)
-			abi, err := interfaceGetter.GetABI(constants.PublicResolverAddress, blockNumber)
-			Expect(err).NotTo(HaveOccurred())
+			abi := interfaceGetter.GetABI(constants.PublicResolverAddress, blockNumber)
```

Comment:
Didn't I recently add this error propagation? If running the tests without the mainnet infura key set, the error is very confusing. This passed along the error from geth instead of making that `abi` just the empty string.

Reply:
My mistake, that makes sense- reverting back.